### PR TITLE
Add Application project and fix bugs

### PR DESCRIPTION
**Pull request by Boburm1rzo**

### CODE REVIEW

File: Web/Ecommerce/Ecommerce.Application/Mappings/CategoryMappings.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public static CategoryViewModel ToViewModel(this Category category)
+    {
+        return new CategoryViewModel
+        {
+            Id = category.Id,
+            Name = category.Name,
+            Description = category.Description,
+        };
+    }
+    public static UpdateCategoryViewModel ToUpdateViewModel(this CategoryViewModel category)
```
Comment (Ramadan1011) [inline Web/Ecommerce/Ecommerce.Application/Mappings/CategoryMappings.cs:17]: must be an empty line

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs [hunk 2]
```diff
@@ -1,12 +1,13 @@
-﻿using Ecommerce.ViewModels.PaymentDetail;
+﻿using Ecommerce.Application.ViewModels.PaymentDetail;
+using Ecommerce.Domain.Entities;
 
-namespace Ecommerce.Mappings
+namespace Ecommerce.Application.Mappings
 {
```
Comment (Ramadan1011) [inline Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs:5]: image
Convert file scoped namespace

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs [hunk 3]
```diff
@@ -15,7 +16,7 @@ public static ViewModels.PaymentDetail.PaymentDetailViewModel ToViewModel(this D
                 OrderId = paymentDetail.OrderId,
             };
         }
-        public static UpdatePaymentDetailViewModel ToUpdateViewModel(this ViewModels.PaymentDetail.PaymentDetailViewModel paymentDetail)
+        public static UpdatePaymentDetailViewModel ToUpdateViewModel(this PaymentDetailViewModel paymentDetail)
```
Comment (Ramadan1011) [inline Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs:19]: must be an empty line.

File: Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs [hunk 4]
```diff
@@ -1,7 +1,7 @@
-﻿using Ecommerce.Domain.Entities;
-using Ecommerce.ViewModels.Product;
+﻿using Ecommerce.Application.ViewModels.Product;
+using Ecommerce.Domain.Entities;
 
-namespace Ecommerce.Mappings
+namespace Ecommerce.Application.Mappings
```
Comment (Ramadan1011) [inline Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs:4]: convert file scoped namespace

File: Web/Ecommerce/Ecommerce.Application/ViewModels/Category/CategoryViewModel.cs [hunk 5]
```diff
@@ -1,9 +1,11 @@
-﻿namespace Ecommerce.ViewModels.Category
+﻿
+namespace Ecommerce.Application.ViewModels.Category
 {
```
Comment (Ramadan1011) [inline Web/Ecommerce/Ecommerce.Application/ViewModels/Category/CategoryViewModel.cs:3]: convert file scoped namespace all files

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs [hunk 6]
```diff
@@ -26,21 +27,20 @@ public static UpdatePaymentDetailViewModel ToUpdateViewModel(this ViewModels.Pay
                 OrderId = paymentDetail.OrderId,
             };
         }
```
Comment (jamshidbek7375) [inline Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs:29]: did you add a line?

File: Web/Ecommerce/Ecommerce/Services/PaymentDetailService.cs [hunk 7]
```diff
@@ -1,14 +1,15 @@
...
 using Ecommerce.Services.Interfaces;
-using Ecommerce.ViewModels.PaymentDetail;
 
 namespace Ecommerce.Services;
 
 public class PaymentDetailService : IPaymentDetailService
 {
     private readonly ICommonRepository _commonRepository;
 
+
```
Comment (Ramadan1011) [inline Web/Ecommerce/Ecommerce/Services/PaymentDetailService.cs:12]: why did you add an empty line

**Review by jamshidbek7375 — APPROVED** [review]
You will unexpectedly find time and work well

**Review by Ramadan1011 — CHANGES_REQUESTED** [review]
Fix the code I commented.

File: Web/Ecommerce/Ecommerce.Application/Mappings/CategoryMappings.cs (hunk 1 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce.Application/Mappings/CategoryMappings.cs:17]: where did you see an empty line?

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs (hunk 2 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs:5]: I wonder if the screenshot is from another project
sorry but i dont understand :(

File: Web/Ecommerce/Ecommerce.Application/ViewModels/Category/CategoryViewModel.cs (hunk 5 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce.Application/ViewModels/Category/CategoryViewModel.cs:3]: I don't understand where there is a mistake

File: Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs (hunk 4 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs:4]: I don't understand where there is a mistake

File: Web/Ecommerce/Ecommerce/Services/PaymentDetailService.cs (hunk 7 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce/Services/PaymentDetailService.cs]: fixed

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs (hunk 6 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs]: sorry but I don't understand

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs (hunk 2 above)
Comment (Ramadan1011) [reply Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs]: Yes it's a different project and it has a compiler error.
A semicolon should be at the end of the namespace line;

File: Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs (hunk 4 above)
Comment (Ramadan1011) [reply Web/Ecommerce/Ecommerce.Application/Mappings/ProductMappings.cs:4]: I think, you got me

File: Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs (hunk 2 above)
Comment (Boburm1rzo) [reply Web/Ecommerce/Ecommerce.Application/Mappings/PaymentDetailMappings.cs]: I understand now

**Review by Ramadan1011 — APPROVED** [review]
nice! keep going bro
